pkgs/development/haskell-modules/all-cabal-hashes.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Is there anyway we could get this file (and `pkgs/development/haskell-modules/cabal2nix-latest.json`) in YAML, and have a descriptive comment at the top explaining what they are for, how they are used, etc?

It is easy enough for us to guess what the file is for based on its filename and contents, but I imagine it is quite a bit harder for new users.